XMTP.podspec

```diff
@@ -22,6 +22,7 @@ Pod::Spec.new do |spec|
   spec.frameworks 		= "CryptoKit", "UIKit"
 
   spec.dependency 'CSecp256k1', '~> 0.2'
+  #TODO: spec.dependency 'secp256k1.swift', '~> 0.1'
```

Comment:
Can't bring this back. Would also cause issues for out integrators if we changed our podspec would prefer if we just removed this todo